3기/choijiyoung/week2/css/style.css

```
@@ -0,0 +1,167 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;700;900&display=swap');
+@font-face {
+  font-family: 'Pretendard-Regular';
+  src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff')
+    format('woff');
+  font-weight: 400, 700;
+  font-style: normal;
+}
+
+:root {
+  --primary: #2d3436;
+  --secondary: #636e72;
+  --coral: #ff7f50;
+  --white: #fff;
+  --cityLight: #dfe4ea;
+}
+
+*,
+::before,
+::after {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Inter', 'Pretendard-Regular', sans-serif;
+  font-size: 16px;
+  line-height: 160%;
+}
+
+button {
+  border: none;
+  outline: none;
+  background: none;
+}
```

디자인적으로 추가 UI가 들어간 것이 아니라면, 접근성을 위해서 버튼에 outline 은 살아있어야해요!
그래야 사용자가 텝으로 사이트를 이동할 때, 내 텝의 포커스가 어디에 있는지 알 수 있어요.

```suggestion
button {
  border: none;
  background: none;
}
```